src/charts/vertical_bar.coffee

```diff
@@ -44,6 +44,38 @@ Ember.Charts.VerticalBarComponent = Ember.Charts.ChartComponent.extend(
   # Data
   # ----------------------------------------------------------------------------
 
+  # Override sortedData from SortableChartMixin
+  sortedData: Ember.computed ->
+    if @get 'stackBars'
+      data = @get 'data'
+      groupedData = _.groupBy data, (d) -> d.group
+      summedGroupValues = []
+      for group, groupData of groupedData
+        if group isnt "null" and group isnt null
```

Comment:
yeah it's odd, there must've been cases where group was either null or "null" I forget